package.json

```diff
@@ -22,6 +22,8 @@
   },
   "devDependencies": {
     "LiveScript": "latest",
+    "typescript": "latest",
```

Comment:
Looks like `typescript-compiler` already includes `typescript` as a git submodule so I don't think this line is needed.